views/assets/javascripts/tyk.js

```diff
@@ -11,3 +11,14 @@ function onCancel() {
 	}
 }
 
+document.getElementById("profile-icon").addEventListener("mouseover", showProfileMenu);
```

Comment:
@ConsM the code in this file might not be needed and a pure CSS solution might work. Even so, if you get to write code like this, you could try to organised it better even if it's a small piece of functionality

Here's how I would've done it

```
//profile-menu.js
function ProfileMenu ({id}) {
  const $element = document.getElementById(id);

  const toggleMenu = (action) => {
    $element.style.visibility = action;
  }

  const show = () => {
    toggleMenu("visible");
  }

  const hide = () => {
    toggleMenu("hidden");
  }

  return {
    show,
    hide
  }
};

export default ProfileMenu;

//tyk.js

import ProfileMenu from './profile-menu';

const profileMenu = ProfileMenu({id: "profile-menu"});

document.getElementById("profile-icon").addEventListener("mouseover", profileMenu.show);
document.getElementById("logout-btn").addEventListener("mouseout", profileMenu.hide);
```